### PR TITLE
data: pritamrajput

### PR DESCRIPTION
**Pull request by pritamrajput**
## Check List (Check all the applicable boxes) 

- [ ] My code follows the code style of this project.
- [ ] My change requires changes to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] All new and existing tests passed.
- [ ] This PR does not contain plagiarized content.
- [ ] The title of my pull request is a short description of the requested changes.

## Note to reviewers

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/4039"></a>

### CODE REVIEW

**Review by pritamrajput — COMMENTED** [review]
what should I do now ?

**Comment by pritamrajput** [comment]
JSON check is not getting passed. How to make changes and How should I proceed Now ?

File: data/pritamrajput.json [hunk 1]
```diff
@@ -21,4 +21,6 @@
       "icon": "FaGithub"
     }
   ]
```
Comment (kumarsonsoff3) [inline data/pritamrajput.json:23]: Please add a missing comma `,` here to validate your json! 

```suggestion
  ],
```

**Review by kumarsonsoff3 — CHANGES_REQUESTED** [review]

File: data/pritamrajput.json (hunk 1 above)
Comment (pritamrajput) [reply data/pritamrajput.json:23]: Okay and than create new pull request ?

Comment (kumarsonsoff3) [reply data/pritamrajput.json]: No-no, you did it! 🎉

**Review by kumarsonsoff3 — APPROVED** [review]
Looks good! 🎉

**Comment by pritamrajput** [comment]
How can I access my updated linkfree link ?

**Comment by kumarsonsoff3** [comment]
> How can I access my updated linkfree link ?

Just go to your linkfree URL which should be https://linkfree.eddiehub.io/pritamrajput, changes will be displayed once deployed!

**Pull request merged**